validator/sql_public_ip.rego

```diff
@@ -0,0 +1,16 @@
+package templates.gcp.SQLPUBLICIPV1
+
+import data.validator.gcp.lib as lib
+
+deny[{
+	"msg": message,
+	"details": metadata,
+}] {
+	asset := input.asset
+	asset.asset_type == "sqladmin.googleapis.com/Instance"
+
+	asset.resource.data.settings.ipConfiguration.ipv4Enabled == true
```

Comment:
Is the `ipConfiguration` object always present?

Reply:
Yes